### ISSUE TITLE
Wallet UI improvments needed, Rework the Forms

### ISSUE THREAD

**Issue by adityakm24** [adityakm24/SynthiFy#3]

**Comment by IAmRiteshKoushik** [comment]
Is this issue still being worked upon or has it been resolved ?

**Comment by adityakm24** [comment]
Nope this is still being worked upon, it is active

**Comment by IAmRiteshKoushik** [comment]
If there is a bit of a detailed roadmap on the things that need to be implemented / improved; say a couple of different issues, then I would like to contribute 👋

**Comment by adityakm24** [comment]
sure what are you expecting? for now UI needs improvements, as in it needs to be responsive. We are coming up with a roadmap soon.

**Comment by 0xzrf** [comment]
I would like to work on this. I think I can make it more responsive, can I use external libraries for this(Like MUI)?
They provide Grid and Stacks for responsiveness across multiple screen-sizes

**Comment by adityakm24** [comment]
Yes can you please check out the UI-new branch: https://github.com/adityakm24/SynthiFy/tree/UI-new
